pom.xml

```diff
@@ -1415,6 +1415,11 @@
         <artifactId>spring-jdbc</artifactId>
         <version>${spring.version}</version>
       </dependency>
+      <dependency>
```

Comment:
atlas-intg has a transitive dependency on spring-context.
spring-context was bringing in the older version (5.3.21), as it wasn't included in the POM files prior. 
Since this dependency has multiple occurrences, it is better to add it explicitly as a dependency.

Reply:
Based on the comment on Line 228, https://github.com/apache/hive/blob/1d7b490a644f87ead2997387e947f26f8de34324/pom.xml#L228,
have you tried upgrading atlas as well?

Reply:
The current version of atlas in use, 2.3.0 is already the latest one, hence this wasn't possible.

Reply:
> atlas-intg has a transitive dependency on spring-context. spring-context was bringing in the older version (5.3.21), as it wasn't included in the POM files prior. Since this dependency has multiple occurrences, it is better to add it explicitly as a dependency.

Excluding **spring-context** from **atlas-intg** in **ql/pom.xml** successfully removed the older versions being called. Therefore, instead of including as an additional dependency, chose to exclude it instead.

Reply:
> @arorasimran0309 Since the spring-context jar itself will be excluded in the lib post this, there will not be any impact during runtime ?

In atlas-intg, spring-context is only used by AtlasTypeRegistry, which is neither a direct nor indirect dependency in Hive.